Dockerfile

```diff
@@ -20,12 +20,11 @@ RUN rm -rf ${ANDROID_DIR}/tools \
 # Required to accept licenses:
 # https://stackoverflow.com/questions/38096225/automatically-accept-all-sdk-licences/42125740#42125740
 # There will be a bit of delay (approx 30 seconds)
-RUN yes | ${ANDROID_DIR}/tools/bin/sdkmanager "platform-tools" "platforms;android-23" "build-tools;23.0.2" "extras;android;m2repository" "extras;google;m2repository"
+RUN yes | ${ANDROID_DIR}/tools/bin/sdkmanager "platform-tools" "platforms;android-26" "build-tools;26.0.3" "extras;android;m2repository" "extras;google;m2repository"
```

Comment:
I would change this to `platforms;android-27` and `build-tools;27.0.3` because that will be the target very soon.

Reply:
Good idea, this PR is dependent on your other changes anyways so might as well make it the latest.